spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -611,19 +671,23 @@ def set_current_editor(self, editor, update, clear):
         
     def remove_editor(self, editor):
         self.treewidget.remove_editor(editor)
-        
+
     def get_options(self):
         """
         Return outline explorer options
         """
-        return dict(show_fullpath=self.treewidget.show_fullpath,
-                    show_all_files=self.treewidget.show_all_files,
-                    group_cells=self.treewidget.group_cells,
-                    show_comments=self.treewidget.show_comments,
-                    expanded_state=self.treewidget.get_expanded_state(),
-                    scrollbar_position=self.treewidget.get_scrollbar_position(),
-                    visibility=self.isVisible())
-    
+        return dict(
+            show_fullpath=self.treewidget.show_fullpath,
+            show_all_files=self.treewidget.show_all_files,
+            group_cells=self.treewidget.group_cells,
+            show_comments=self.treewidget.show_comments,
+            sort_files_alphabetically=

```

Comment:
I do not know how to fix the two pep8 issues related to these two lines elegantly. Please help.

Reply:
Its just

```python
sort_files_alphabetically=(
    self.treewidget.sort_files_alphabetically)
```

Reply:
Cool, thanks